### ISSUE TITLE
Postgres.yaml is missing for local test deployment ...

### ISSUE THREAD

**Issue by DeepDiver1975** [InfiniteDevices/infinimesh#51]
**To Reproduce**
bash <(curl -s https://raw.githubusercontent.com/infinimesh/infinimesh/master/hack/microk8s/infinimesh-setup-ubuntu.sh)
and after some time:

```
=> now we install infinimesh ...

error: unable to read URL "https://raw.githubusercontent.com/infinimesh/infinimesh/master/hack/microk8s/postgres.yaml", server reported 404 Not Found, status code=404
platform.infinimesh.infinimesh.io/my-infinimesh created
kafka.kafka.strimzi.io/infinimesh created
 checking for host entries

```

**Comment by DeepDiver1975** [comment]
Later there is another error - most likely related sue to the missing postgres:

```
wait a few secs to get all things in place ....
1
infinimesh is now ready, point your browser to app.infinimesh.local or use our CLI

your master user credentials are: 
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
root
W0720 15:43:14.101627    5943 loader.go:223] Config not found: /home/deepdiver/kubeconfig
Error from server (NotFound): secrets "my-infinimesh-root-account" not found

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

**Comment by DeepDiver1975** [comment]
Finally: what could the password be? ;-)

**Comment by 2pk03** [comment]
fixed in fae236f:
https://github.com/InfiniteDevices/infinimesh/commit/fae236fc69de313847c003664101a7b361ea24ec

thanks for reporting!
